File: generator/output/ts-model/app-instance.model.ts
class AppInstanceModel {
  public id: number = 0;

  // deleted 是否删除
  public deleted: boolean = false;

  // auto_save 是否删除
  public auto_save: boolean = false;

  // status 实例状态 Pending | Starting | Running ｜ Stopping
  public status: string;

  // image_id 镜像id
  public image_id: string;

  // application_id APPid
  public application_id: number = 0;

  // public_address 公有地址
  public public_address: string;

  // origin_image_id 原有镜像id
  public origin_image_id: string;

  // instance_type 实例规格
  public instance_type: string;

  // hours 使用时长
  public hours: number = 0;

  // password 实例id
  public password: string;

  // private_address 私有地址
  public private_address: string;

  // user_id 用户id
  public user_id: number = 0;

  // instance_id 实例id
  public instance_id: string;

  public created_at: string;

  public updated_at: string;
}

const AppInstanceModelFields = {
  id: "id",
  deleted: "deleted",
  auto_save: "auto_save",
  status: "status",
  image_id: "image_id",
  application_id: "application_id",
  public_address: "public_address",
  origin_image_id: "origin_image_id",
  instance_type: "instance_type",
  hours: "hours",
  password: "password",
  private_address: "private_address",
  user_id: "user_id",
  instance_id: "instance_id",
  created_at: "created_at",
  updated_at: "updated_at"
};

export {AppInstanceModel, AppInstanceModelFields};

File: generator/output/entityModel/AppInstanceModel.java
package com.licenseair.backend.domainModel;

import java.util.List;
import lombok.Data;

import java.lang.String;
import java.lang.Long;
import java.lang.Boolean;
import java.lang.Integer;
import java.sql.Timestamp;

/**
* Created by licenseair.com
* AppInstance
*/
@Data()
public class AppInstanceModel {
  public Long id = null;

  /**
  * 是否删除
  */
  public Boolean deleted = null;

  /**
  * 是否删除
  */
  public Boolean auto_save = null;

  /**
  * 实例状态 Pending | Starting | Running ｜ Stopping
  */
  public String status = null;

  /**
  * 镜像id
  */
  public String image_id = null;

  /**
  * APPid
  */
  public Long application_id = null;

  /**
  * 公有地址
  */
  public String public_address = null;

  /**
  * 原有镜像id
  */
  public String origin_image_id = null;

  /**
  * 实例规格
  */
  public String instance_type = null;

  /**
  * 使用时长
  */
  public Integer hours = null;

  /**
  * 实例id
  */
  public String password = null;

  /**
  * 私有地址
  */
  public String private_address = null;

  /**
  * 用户id
  */
  public Long user_id = null;

  /**
  * 实例id
  */
  public String instance_id = null;

  public Timestamp created_at = null;

  public Timestamp updated_at = null;

}
